### ISSUE TITLE
meshcat: "invalid escape sequence"

### ISSUE THREAD

**Issue by EricCousineau-TRI** [RobotLocomotion/drake#10526]
From this test (the top-level bit):
https://drake-cdash.csail.mit.edu/testDetails.php?test=25858064&build=1110775

```
meshcat_python/src/meshcat/transformations.py:1042: DeprecationWarning: invalid escape sequence \*
  """
```
Reproducible via:
```
$ bazel run --config=python3 //bindings/pydrake:py/all_test
```

**Comment by jamiesnape** [comment]
https://github.com/rdeits/meshcat-python/blob/d0c8b6a9d1d750495ef9513254761cc14773cf99/src/meshcat/transformations.py#L1001

Can you file an upstream bug?

**Comment by EricCousineau-TRI** [comment]
@jamiesnape Now that this is updated, should we update the fork?

**Comment by jamiesnape** [comment]
Indeed. There are a whole host of other SHA bumps in PR at the moment, so makes sense.

**Comment by jamiesnape** [comment]
You may want to catch #10575 before it merges.

**Comment by jwnimmer-tri** [comment]
I considered that, but it's already passed all CI.  Since you'll need a macOS CI spin anyway for this bump, we might as well do that in a new PR.

**Comment by jwnimmer-tri** [comment]
(I could still roll it into #10576 or #10578 though, if you want to wait on those being ready.)

**Comment by jamiesnape** [comment]
I think a separate PR is fine. I though #10575 was updating it already to a different SHA, but that was `meshcat`, not `meshcat_python`.
